db/c.cc

```diff
@@ -1145,6 +1145,34 @@ rocksdb_column_family_handle_t** rocksdb_create_column_families(
   return c_handles;
 }
 
+rocksdb_column_family_handle_t** rocksdb_create_column_families_with_options(
```

Comment:
I approve of changing the parameter order so as not to allow confusion of options[] and options* ...

Reply:
Sorry @alanpaxton, I don't understand what are you saying. Can you please describe it more?

Reply:
There are different `rocksdb_create_column_famil[y|ies]` methods with different orders of parameters. I was confused by the different orders of parameters. But I think your function has the "correct" order of parameters for what it does.